helix-term/Cargo.toml

```
@@ -70,6 +69,11 @@ grep-searcher = "0.1.11"
 signal-hook-tokio = { version = "0.3", features = ["futures-v0_3"] }
 libc = "0.2.146"
 
+[target.'cfg(not(target_os = "macos"))'.dependencies]
+crossterm = { version = "0.26", features = ["event-stream"] }
+[target.'cfg(target_os = "macos")'.dependencies]
+crossterm = { version = "0.26", features = ["event-stream", "use-dev-tty"] }
```

you don't need the `cfg(not)` case here cargo will automatically unify features (you ofcourse need to keep the existing dependency unchanged them

```suggestion
[target.'cfg(target_os = "macos")'.dependencies]  
crossterm = { version = "0.26", features = ["use-dev-tty"] }
```